Enterprise-CMCS/macpro-mdct-mcr
# YoY frontend update

### Description

The frontend portion of the YoY Copy feature. 
- Once a report has been submitted, it should show up in the "Select which report you'd like to copy" dropdown, as an option of a report to copy (in the New Program modal)
- Choosing to copy from a previous report or not, creating a new program should work as normal.
- If you copy from a previous report, all answers except "VII: Quality Measures" (we are still waiting on the final list of what should and should not be copied over - hence the feature flag)
- 
<img width="533" alt="Screenshot 2023-08-14 at 4 54 58 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mcr/assets/10237149/790a1a49-f62f-462a-a937-9f648045da4d">

### Related ticket(s)


MDCT-2768, MDCT-2771

---
### How to test

- create a new program as normal (no copying)
- submit that program
- create another program, the previous program should show up in the dropdown menu
- choose it and create the new program
- skip to the end to Review & Submit, and see that only the "VII: Quality Measures" option still requires attention, and all the other questions have answers


### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
